velox/core/QueryConfig.h

```diff
@@ -906,6 +912,12 @@ class QueryConfig {
     return get<double>(kTableScanScaleUpMemoryUsageRatio, 0.7);
   }
 
+  int32_t shuffleCompressionKind() const {
```

Comment:
https://github.com/facebookincubator/velox/blob/main/velox/core/QueryConfig.h#L717

Do we also use string as this config? User will not need to know which enum value means the codec. Spark also use string for compression codec.